releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -4,6 +4,22 @@
 # Microsoft.FeatureManagement.AspNetCore
 [Source code ][source_code_web] | [Package (NuGet)][package_web] | [Samples][samples_web] | [Product documentation][docs]
 
+## 3.1.0 - November 23, 2023
+
+### Changes
+
+* Exposing FeatureManager and ConfigurationFeatureDefinitionProvider to public.
```

Comment:
The release notes aren't documentation, so I would be weary of writing something too long here. If it is a long description and you feel it is needed then I would consider one of the two options:

* Write the description in the PR that added it and link to the PR (as you have done).
* Update the readme in the repo.

For release notes I would expect something more succint like
* `FeatureManager` and `ConfigurationFeatureDefinitionProvider` are now public
  * Enables usage of external dependency injection containers
  * Allows usage of feature manager without requiring dependency injection